### PR TITLE
Define and activate AWS 9.3.6 platform release

### PR DESCRIPTION
**Pull request by sslavic**
- [x] Waiting for updated aws-operator that can run latest Flatcar container linux for CVE fixes

### CODE REVIEW

**Comment by sslavic** [comment]
Tested on gauss:
- 9.3.6 cluster creation
- upgrade from 9.3.5

File: aws/v9.3.6/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Updated kube-state-metrics version from 1.9.5 to 1.9.7. Check the [upstream changelog](https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md#v197--2020-05-24) for details on all changes.
+
+### metrics-server v0.3.6 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/metrics-server-app/blob/master/CHANGELOG.md#111---2020-07-23))
+
+- Updated metrics-server version from 0.3.3 to 0.3.6. Check the [upstream changelog](https://github.com/kubernetes-sigs/metrics-server/releases) for details on all changes.
+
+### nginx-ingress-controller v0.34.1 ([Giant Swarm app v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28))
+
+- Drop support for deprecated configuration properties.
```
Comment (cokiengchiara) [inline aws/v9.3.6/README.md:23]: @sslavic might this break anything for customers? I only noticed the dropping of support for configs. How do we know if customers are still using these deprecated configs in their workflow?

**Comment by calvix** [comment]
AWS operator 5.7.2 with latest flatcar  is done,  changes pushed to release file as well

File: aws/v9.3.6/README.md (hunk 1 above)
Comment (cokiengchiara) [reply aws/v9.3.6/README.md:23]: thank you!

**Comment by cokiengchiara** [comment]
> AWS operator 5.7.2 with latest flatcar is done, changes pushed to release file as well

@calvix thank you! Can you also update changelog and release notes? (just w the facts) :)

**Comment by calvix** [comment]
I will now work on the calico and than update release notes

**Comment by calvix** [comment]
in nginx, i see the same error as in 9.0.9
```
E0731 12:30:18.143841       8 leaderelection.go:356] Failed to update lock: configmaps "nginx-ingress-controller-leader-nginx" is forbidden: User "system:serviceaccount:kube-system:nginx-ingress-controller" cannot update resource "configmaps" in API group "" in the namespace "kube-system"
```

**Comment by calvix** [comment]
my changes are done, release notes updated, you can go ahead @sslavic 

I tested cluster creation 9.0.9 and cluster update from 9.0.8

**Comment by sslavic** [comment]
> I tested cluster creation 9.0.9 and cluster update from 9.0.8

@calvix is it copy paste error?

**Comment by sslavic** [comment]
Working on the fix for https://github.com/giantswarm/releases/pull/378#issuecomment-667096073 in giantswarm/nginx-ingress-controller-app#105

File: aws/v9.3.6/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# :zap: Giant Swarm Release v9.3.6 for AWS :zap:
+
+This release updates managed apps to the latest releases, updates Calico to version `3.13.4` and Container Linux to `2512.2.1`.
```
Comment (cokiengchiara) [inline aws/v9.3.6/README.md:3]:
```suggestion
This release updates managed apps to the latest releases, updates Calico to version `3.13.4`, and Flatcar Container Linux to `2512.2.1`.
```

**Comment by sslavic** [comment]
Included nginx leader election fix. Retested on gauss:
- 9.3.6 cluster creation
- upgrade from 9.3.5

**Pull request merged**